nrm_django/grants/forms.py

```diff
@@ -18,10 +26,26 @@ class Meta:
         }
 
     def __init__(self, *args, **kwargs):
-        """A quick init to add some classes on some fields, since we don't need to override the widget."""
+        """
+        A quick init to add some classes on some fields, since we don't need to override the widget,
+        and add a custom field or two.
+        """
         super(GrantForm, self).__init__(*args, **kwargs)
         self.fields["proj_title"].widget.attrs["class"] += " text-wide"
 
+        # If we're modifying an existing instance in a changeform we'll need to set the initial value
+        # for project_category, because we don't have a proper FK from Grant
+        # Also, because we used distinct() to toss out duplicate Cats, we have to find one that
+        # matches whatever one was set on the current Grant so it looks right.
+        if self.instance:
+            this_cat = Category.objects.get(grant=self.instance)
+            match = self.fields["project_category"].queryset.get(
+                category_desc=this_cat.category_desc
+            )
+
+            print(match.cn)
```

Comment:
this outputs '670010602' but in source I see 
![Screen Shot 2021-04-15 at 12 13 35 PM](https://user-images.githubusercontent.com/32077682/114910902-0bbd6d00-9de4-11eb-97b7-fbc6faa0cf88.png)